src/Pact/Compile.hs

```diff
@@ -158,6 +169,324 @@ currentModule i = use csModule >>= \m -> case m of
   Just cm -> return cm
   Nothing -> syntaxError i "Must be declared within module"
 
+pattern ELitName :: Text -> Exp
```

Comment:
This is a dupe of `ELitString`. Also we wanted to keep the property parsing out of Parse.hs. Please migrate the below stuff to something in Pact/Analyze(/Parse.hs?)